packages/employer-api/src/controllers/claim.controller.ts

```diff
@@ -116,6 +116,54 @@ export const createClaim = async (req: any, res: express.Response) => {
     }
 }
 
+export const createLegacyClaim = async (req: any, res: express.Response) => {
+    try {
+        const bceid_guid = req.kauth.grant.access_token.content?.bceid_user_guid
+        if (bceid_guid === undefined) {
+            return res.status(403).send("Not Authorized")
+        }
+        if (!req.body?.guid || req.body.guid !== bceid_guid) {
+            return res.status(403).send("Forbidden")
+        }
+
+        const prefillFields = {
+            isLegacy: true,
+            container: {
+                businessAddress1: req.body.address,
+                employerCity: req.body.city
+            }
+        }
```

Comment:
Should we require the postal code for validation like we do in the application forms, then prefill the postal code as well? It threw me off a little to have to enter the postal code in the form. Still works find like this though.

Reply:
The validation in the application forms doesn't actually use the postal code at all, which is why I left it out